solr/solrj/src/java/org/apache/solr/common/params/CombinerParams.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.common.params;
+
+/** Defines the request parameters used by all combiners. */
+public interface CombinerParams {
+  String COMBINER = "combiner";
+
+  String COMBINER_ALGORITHM = COMBINER + ".algorithm";
+
+  String COMBINER_KEYS = COMBINER + ".keys";
```

Comment:
WDYT of instead do `combine.q` multi-valued, so just supply each query as another value?  One less indirection and thing to name.

Reply:
mmmmm, I do believe it can be useful to have a "query key", especially because the debug component explains both the query parsing and combination:

0.032522473 = 
1/(60+1) + 1/(60+2) because its ranks were: 1 for query(lexical), 2 for query(lexical2)

So the keys help in my opinion.
If you were referring to this specific parameter, this is not intended for the user, it gets used automatically by Solr.

Happy to elaborate!

Reply:
> ... `combine.q` multi-valued ...

In the case of multi-valued parameters, is ordering 'guaranteed' or could `&combine.q=Foo&combine.q=Bar` turn into `&combine.q=Bar&combine.q=Foo` and if it did would it matter?

Reply:
AFAIK it's guaranteed; we use a List not a Set intentionally